samples/unpeelingOnion/src/onion/ecommerce/application/shoppingCarts/commands/shoppingCarts/confirmShoppingCart.ts

```diff
@@ -0,0 +1,7 @@
+import { Command } from '#core/commands';
+
+export class ConfirmShoppingCart extends Command {
```

Comment:
Command should be an object, so use interface.

Reply:
This example is “the wrong way” 🙂